File: _BHCVRP_Python_Version/cujae_inf_citi_om/exceptions/RequestException.py
class RequestException(Exception):
    def __init__(self, request=None):
        if request is not None:
            self.request = request
        
    def __init__(self, message=None):
        if message is not None:
            super().__init__(message)

    def get_request(self):
        return getattr(self, 'request', None)

File: BHCVRP_Java_Version/src/cujae/inf/citi/om/exceptions/RequestException.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package cujae.inf.citi.om.exceptions;

/**
 *
 * @author nanda
 */

public class RequestException extends Exception{
    private float request;
    
    public RequestException(float request){
        this.request = request;
    
    }
    
    public float getRequest(){
        return request;
    }
    
    public RequestException(String message){   
        super(message);
    }
    
}
